lambda-utils/lib/lambda-utils.ts

```diff
@@ -8,6 +8,21 @@ const logger = new Logger('lambda-utils');
 /** Define the async version of ProxyHandler */
 export type AsyncProxyHandler = (event: APIGatewayEvent, context: Context) => Promise<any>;
 
+/**
+ * Casted interface for APIGatewayProxyEvents converted through the middleware
+ *
+ * @typedef {Interface}
+ * @property {any} body The cast JSON -> object body element
```

Comment:
We don't need these tags - they are useless. Typescript removes the need for some JsDoc that violates the DRY principle, and `body` is being document by its own JsDoc.